options/pin.go

```diff
@@ -17,13 +27,19 @@ type PinUpdateSettings struct {
 	Unpin bool
 }
 
+// PinAddOption pin add option func
```

Comment:
These (and several below) could be a bit more descriptive. I lack familiarity with this library and this does not help give me any.